scripts/release/packageTypes.ts

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
+import { BASE, PACKAGE_TYPES } from './constants';
+
+const integrationWrapperPackages = [
+  'api-client',
+  'composables',
+  'theme'
+];
+
+const getDirectories = (source: string) =>
+  fs.readdirSync(`${BASE}/${source}`, { withFileTypes: true })
+    .filter(dirent => dirent.isDirectory())
+    .map(dirent => dirent.name);
+
+const isPackage = (pckg: string) => fs.existsSync(`${BASE}/${pckg}/package.json`);
+
+const isIntegrationWrapper = (pckg: string) => {
+  try {
+    const directories = getDirectories(pckg);
+    return integrationWrapperPackages.length <= directories.length && integrationWrapperPackages.every(directory => isPackage(`${pckg}/${directory}`));
```

Comment:
please `integrationWrapperPackages.every(directory => isPackage(`${pckg}/${directory}`)` move to the separate function and call it maybe `hasPackage(pckg)` or something because this line is really loong (hard do read it)

Reply:
Sure

Reply:
I just created `hasIntegrationWrapperPackages` function